charts/registry/Chart.yaml

```diff
@@ -32,7 +32,7 @@ appVersion: 0.3.17
 dependencies:
```

Comment:
you need to increase the helm version also. https://github.com/eclipse-tractusx/sldt-digital-twin-registry/blob/main/charts/registry/Chart.yaml#L29

Reply:
> you need to increase the helm version also. https://github.com/eclipse-tractusx/sldt-digital-twin-registry/blob/main/charts/registry/Chart.yaml#L29

Thanks for the review comments @agg3fe . I have increased the helm version.